### ISSUE TITLE
Node staking reward distribution shorter interval (daily option)

### ISSUE THREAD

**Issue by CryptoTurkish** [StorXNetwork/StorX-Node#5]
Dear Team,

I would like to request we have the interval reduced from 30 days to daily to give a consistent smooth release instead of huge supply shock at 30 days interval where every node dumps huge amounts each month. Many other projects distribute rewards daily as well.

Thanks!
CT

**Comment by DrSvett** [comment]
That would be awesome. i totally agree with the writer above!

**Comment by Cryptogateway2020** [comment]
Slowly distribution per block or day or week would be good for the project.  It will help avoiding the selling pressure at 1 given time.  Or giving people the chance to claim reward anytime they like to.
Some projects distribute reward live per block, and it is up to node owners to claim it or not.
* strongblock
* Trava
* Orai .....
* Just name a few.
Thank you team.

**Comment by ajnozari** [comment]
Honestly this is a good idea in principle.  It serves several groups while helping the overall inflation problem. However it comes with one downside to new operators who aren’t already a part of the network, but I’ll get to that.

First and foremost it will slow node surges, this will help with network stability and allows the growth to be more continuous rather than jumping monthly.

This has the added benefit of slowing the strain on vps providers.  More predictable costs on their side means more stable costs for node operators as well. Overall both sides benefit from a more relaxed rollout.

It will raise the cost to stake faster, but it gives current node operators the chance to add nodes during the month, rather than having to wait to re-invest with the network relying on new investments and new operators coming online solely.  This will overall slow down the inflation as it puts limits on the number of new operators who can come online.

The downside is new operators are disadvantaged as they have to compete with already staked nodes.  However this isn’t a bad thing as it addresses the inflation problem while still allowing fresh cash to flow into the market.

In the end the decision will have to be, is the benefit to the network, it’s stability, and the providers backing it (vps) worth more than the disadvantage it gives to new operators.

I am biased when I say I like this proposal, but I have reasons aside being already staked.  The strain on node operators is a major concern as they may change their limits to compensate.  These limit changes can affect existing and new nodes.

**Comment by Cryptogateway2020** [comment]
Overall, it is matter most is the sustainability and network stability for existing nodes and new nodes.
There can be a way to reduce the tokens by burning them.  I commented on discord.
If we have different type of NFTs, the we can exchange xamount of srx for the nfts.  The NFTs will have different value from srx ( likee 100, 500, 1000 or 5000 srx etc).  These NFTs can be used to stake in the nodes to earn additional rewards ( will have less incentive % wise compared to running a node).  All srx will be burned during the exchange for nfts.

The NFT can maintain the reward rate throughout the year even if the node operator has rewards reduced in the future change.  this is the model the strongblock used and I think it is not bad.  We can help to burn more tokens to help the inflation rate......

Just a thought

**Comment by Swordfish256** [comment]
I a nut shell, the ability for node owners to redeem their reward tokens at any time and ability to add/increse to their stake without the need to unstake would be beneficial and convenient to all parties.

**Comment by Cryptogateway2020** [comment]
Yes, I would like to have option of unstake certain amount of tokens without affecting the node.  I have some nodes with 5000 tokens, so I have to unstake it all to get 5000 to spread to 4 other nodes.  It is a nice option to have.
Thanks

**Comment by TheEqualizing** [comment]
adjusting pay-outs from monthly to weekly would be a huge improvement already.

**Comment by getamar** [comment]
In Gala games, rewards are distributed daily to the treasure chest. If any one wants to redeem the rewards, they pay their own gas fee and mint to their wallet. this way daily distribution gas fee is avoided to the network and passed to the node owners. Currently the XDC gas fee is negligible, hence it is call from the devs to either send the payout daily or enable node owners to redeem when they want (daily or weekly or monthly) and pay the gas fee.
